cli/command/container/create.go

```diff
@@ -233,13 +236,26 @@ func createContainer(ctx context.Context, dockerCli command.Cli, containerConfig
 		return nil
 	}
 
+	var platform *specs.Platform
+	// Engine API version 1.41 first introduced the option to specify platform on
+	// create. It will produce an error if you try to set a platform on older API
+	// versions, so check the API version here to maintain backwards
+	// compatibility for CLI users.
```

Comment:
Should we change https://github.com/docker/cli/blob/590f3271ef098f1e11dfbd84d8596b7bf1561c36/cli/command/utils.go#L125-L129 to restrict the option to API v1.41 and up?

Reply:
Platform is supported on pull since I don't know when.